package.json

```
@@ -16,17 +16,17 @@
   },
   "homepage": "https://github.com/y-scope/eslint-config-yscope#readme",
   "peerDependencies": {
-    "@stylistic/eslint-plugin": "^2.12.1",
-    "eslint": "^9.18.0",
-    "eslint-import-resolver-typescript": "^3.7.0",
-    "eslint-plugin-import": "^2.31.0",
+    "@stylistic/eslint-plugin": "^5.4.0",
+    "eslint": "^9.36.0",
+    "eslint-import-resolver-typescript": "^4.4.4",
+    "eslint-plugin-import": "^2.32.0",
     "eslint-plugin-import-newlines": "^1.4.0",
-    "eslint-plugin-jsdoc": "^50.6.1",
+    "eslint-plugin-jsdoc": "^60.5.0",
     "eslint-plugin-no-autofix": "^2.1.0",
-    "eslint-plugin-react": "^7.37.3",
-    "eslint-plugin-react-hooks": "^5.1.0",
+    "eslint-plugin-react": "^7.37.5",
+    "eslint-plugin-react-hooks": "^5.2.0",
```

seems this was released yesterday
```suggestion
    "eslint-plugin-react-hooks": "^6.1.0",
```